src/application/common/commands-common.ts

```diff
@@ -8,6 +8,7 @@
 import { MAIN_MENU_BAR, MenuContribution, MenuModelRegistry } from './menu';
 import { CommandContribution, CommandRegistry } from './command';
 import { injectable } from "inversify";
+let Globalize = require("globalize");
```

Comment:
it is already used, `globalize` is not es6 module :( and cannot be imported as one